src/foam/nanos/auth/User.js

```diff
@@ -269,14 +234,7 @@ foam.CLASS({
         chooses to be used as a password but may or may not pass as valid.`,
       displayWidth: 30,
       width: 100,
-      storageTransient: true,
-      validateObj: function (password) {
-        var re = /^(?=.*?[A-Z])(?=.*?[a-z])(?=.*?[0-9]).{7,32}$/;
-
-        if ( password.length > 0 && ! re.test(password) ) {
-          return 'Password must contain one lowercase letter, one uppercase letter, one digit, and be between 7 and 32 characters in length.';
-        }
-      }
+      storageTransient: true
```

Comment:
What's the point of desiredPassword vs password?

Reply:
no idea. I'll add this comment to cpf-2259

Reply:
Just as a point of reference, despite having discusses this in person. 

desiredPassword is a string with the desired password, and password stores the current password as a hash of desiredPassword. (desiredPassword is set to null is set on userDAO.put)